packages/scite.rb

```diff
@@ -25,6 +25,8 @@ class Scite < Package
   depends_on 'hicolor_icon_theme'
   depends_on 'shared_mime_info'
   depends_on 'sommelier'
+  depends_on 'libxcursor' 
+  depends_on 'xcb_util_cursor'
```

Comment:
Maybe these should be included in the sommelier package to other apps don't have cursor issues?  In any event, we should be adding new dependencies above sommelier so the postinstall messages don't scroll past the users' visible screen area.

Reply:
okay